### ISSUE TITLE
Hi monsieur, testing the new Records function! Is it true that the function is broken? I uploaded record files (IWF and EWF and Dutch standards) but they won't show up on the scoreboard even if I make a new world record (I wish!)

### ISSUE THREAD

**Issue by jflamy** [jflamy/owlcms4#521]
Hi monsieur, testing the new Records function! Is it true that the function is broken? I uploaded record files (IWF and EWF and Dutch standards) but they won't show up on the scoreboard even if I make a new world record (I wish!)

![image](https://user-images.githubusercontent.com/26678110/188328812-72186ac1-996f-4c0c-8ec2-91342b4c4340.png)

_Originally posted by @JDFS404 in https://github.com/jflamy/owlcms4/issues/517#issuecomment-1236394957_

**Comment by jflamy** [comment]
@JDFS404 this is the new issue I created.

**Comment by jflamy** [comment]
@JDFS404 Please make sure you copied the files under the local/records installation directory BEFORE starting the program (if it was running, you need to restart it).
You should see a message during the startup confirming that the record files have been loaded.

![Fl8bCDW7CH](https://user-images.githubusercontent.com/678663/188329595-c677db23-5bb1-4340-ba47-d3cfb721ff30.png)

**Comment by jflamy** [comment]
@JDFS504 Also make sure that your scoreboard is actually showing the records

![image](https://user-images.githubusercontent.com/678663/188329806-88ba9cec-aa6d-4967-8a0e-678146646c01.png)

**Comment by JDFS404** [comment]
Thanks, I've figured out to restart and saw that this was reading in: 
![image](https://user-images.githubusercontent.com/26678110/188330017-3561f83c-93d0-498a-a4a1-086da123b960.png)

Scoreboard is showing records / category leaders:
![image](https://user-images.githubusercontent.com/26678110/188330051-bc901430-dae5-442e-9f25-0713cd3ee088.png)

107 kg should be a new national record in -67, nothing's happening :(

**Comment by JDFS404** [comment]
Is the folder case sensitive and should it be ```records``` and not `Records`

**Comment by jflamy** [comment]
Yes, it is case sensitive on a Mac.
Attach (drag/drop) your Dutch record file to your reply to this message please.
The messages for reading the files are at the very beginning, just after the program starts.
You are loading a zip I understand.  Make sure that the file are under local/records.

**Comment by jflamy** [comment]
The log files you were seeing are when the zip file is extracted, which would happen before the files are loaded (obviously).
In the latest version ( https://github.com/owlcms/owlcms4-prerelease/releases/tag/34.0.0-beta01 ) I believe I turned off that logging.

**Comment by JDFS404** [comment]
It's working now!
![image](https://user-images.githubusercontent.com/26678110/188330362-c14c6978-74bd-49ce-8a18-100fac35d610.png)
